src/source-nfq.c

```diff
@@ -504,14 +504,7 @@ static void NFQReleasePacket(Packet *p)
 static int NFQBypassCallback(Packet *p)
 {
     if (IS_TUNNEL_PKT(p)) {
-        SCMutex *m = p->root ? &p->root->tunnel_mutex : &p->tunnel_mutex;
-        SCMutexLock(m);
-
-        p->root->nfq_v.mark = (nfq_config.bypass_mark & nfq_config.bypass_mask)
-                        | (p->nfq_v.mark & ~nfq_config.bypass_mask);
-        p->root->flags |= PKT_MARK_MODIFIED;
-
-        SCMutexUnlock(m);
+        return 0;
```

Comment:
Can you add function docs to this function that explains the meaning of the retval?